sale_delivery_split_date/models/sale_order_line.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.tools import format_date
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _prepare_procurement_group_vals(self):
+        vals = super()._prepare_procurement_group_vals()
+        if self._get_procurement_group_key()[0] == 24:
+            if self.commitment_date:
```

Comment:
can you leave the if statement on a single line?

`if self._get_procurement_group_key()[0] == 24 and self.commitment_date:`